package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-demo-payment-service",
-  "version": "2.5.0",
+  "version": "2.5.1",
```

Comment:
For consistency with the other repos should the version increment be to `2.6.0`?

Reply:
updated as suggested